cli/marketplace/build.py

```diff
@@ -687,14 +694,14 @@ def build_temp_docs(temp_root, temp_docs, source_dir):
 
     sphinx_apidoc_cmd(cmd.split(" "))
 
-    shutil.copytree(source_dir / "cli" / "marketplace" / "_static" / "css", temp_docs / '_static/css')
+    shutil.copytree(PROJECT_ROOT / "cli" / "marketplace" / "_static" / "css", temp_docs / '_static/css')
     click.echo("[Sphinx] Done autodoc")
 
 
 if __name__ == "__main__":
     # build_marketplace_cli()
     build_marketplace(
```

Comment:
remove the comment line + I'm not sure how does it work because in L96 you changed the source name to be mandatory and here we don't pass it

Reply:
Also, what will happen with the source_dir when we rename the repo to hub_assets?